scripts/lib/mkcloud-onhost.sh

```diff
@@ -121,7 +121,7 @@ function onhost_cacheclouddata
         for a in $architectures; do
             local suffix
             # Mirror SLES/HA/SOC update + pool repos
-            for suffix in Pool Updates; do
+            for suffix in Pool Updates Updates-test; do
```

Comment:
yes, I suggest to patch the line 129 here:

https://github.com/guangyee/automation/blob/57810478daef57b430571e5a2ade997b7fa910c3/scripts/lib/mkcloud-onhost.sh#L129

something like this should do it:

```bash

[[ $want_test_updates = 1 ]] && {
   [[ $hacloud = 1 ]] && echo "repos/$a/SLE$slesversion-HA-Updates-Test/***"
  
}